test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js

```diff
@@ -88,7 +88,7 @@ define ([
         });
 
         it('Should render properly with a Globus linked account', (done) => {
-            let $node = $('<div>'),
+            var $node = $('<div>'),
```

Comment:
? why the change?

Reply:
Generally I try and avoid using `let` because it can cause unanticipated bugs at times because it's scope acts pretty differently than `var`.